users/vosechu/vosechu.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include "quantum.h"
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+enum userspace_custom_layers {
+  DV = 0,
+  QW,
+  GAM1,
+  RSE,
+  LWR,
+  LFT,
+  MOUSE
+};
+
+enum userspace_custom_keycodes {
+  PAWFIVE = SAFE_RANGE,
+  MOUKEY,
+  MS_BTN1,
+  MS_BTN2,
+  MS_BTN3
+};
+
+// Fillers to make layering more clear
+#define _______ KC_TRNS
+#define _LAYER_ KC_TRNS
+#define XXXXXXX KC_NO
```

Comment:
These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```